data/sql_setup/prepare_schedule_a.sql

```diff
@@ -214,3 +214,16 @@ create trigger nml_sched_a_after_trigger after insert or update
 drop trigger if exists nml_sched_a_before_trigger on disclosure.nml_sched_a;
 create trigger nml_sched_a_before_trigger before delete or update
     on disclosure.nml_sched_a for each row execute procedure ofec_sched_a_delete_update_queues(:START_YEAR_AGGREGATE);
+
+CREATE OR REPLACE FUNCTION insert_sched_master() RETURNS TRIGGER AS $$
+DECLARE
+    child_table_prefix TEXT = TG_ARGV[0];
+    child_table_name TEXT;
+BEGIN
+    child_table_name = child_table_prefix || get_partition_suffix(new.two_year_transaction_period);
```

Comment:
We'll need to derive both the `two_year_transaction_period` and `timestamp` fields (though potentially not `timestamp` - need to double check) before we can just access them an insert them with all of the other values.

I imagine that before this line, we'll need to still declare these like so:

```sql
timestamp TIMESTAMP = CURRENT_TIMESTAMP;
two_year_transaction_period SMALLINT;
```

And then set `two_year_transaction_period` like so:

```sql
two_year_transaction_period = get_transaction_year(view_row.contb_receipt_dt, view_row.rpt_yr);
```

Furthermore, the nature of deriving the value for `two_year_transaction_period` has come up in discussion recently with the FEC DBA folks, so we should also verify this formula (or see if we can/need to just stick solely with `rpt_yr`.  @LindsayYoung, do you remember if there was anything specific said in the last meeting we had with them around this or if we'd still have to investigate further?